src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1307,3 +1307,9 @@
 type: group
 short-summary: Manage resource tags.
 """
+
+helps['version'] = """
+type: command
+short-summary: Show the versions of Azure CLI modules and extensions in JSON format
```

Comment:
The output format can be configured through `-o` option, right?

Reply:
Sure thing!

Reply:
Nice. Maybe replacing `JSON fomat` with `format configured by -o ` in the short-summary would be more clear.

Reply:
Or maybe:

> Show the versions of Azure CLI modules and extensions in JSON format by default or format configured by --output.